README.md

```diff
@@ -29,6 +29,9 @@ Note that on Windows platforms, the ":" character in the classpath must be chang
   * Gradle is the tool used in this repository for automation of the build. Please visit https://gradle.org/ for specific installation instructions.
 
 ### Using Gradle
+Gradle will compile all necessary files and place them in the directory '~/imagelab/build'
```

Comment:
The reference to `~/imagelab/build` appears to be incorrect:
 - `~` is a reference to a user's home directory and only under some system interfaces
 - `imagelab` is directory of the "imagelab" package
 - Not all products are placed in the project's `build` directory (e.g., `imagelab.jar`)